frontend/lib/src/features/home_screen.dart

```diff
@@ -14,6 +17,13 @@ class HomeScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    HomeScreenController homeScreenController = Get.put(HomeScreenController());
+
+    final pages = [
+      Dashboard(),
+      const MyHomePage(title: 'temporary'),
```

Comment:
Edit `MyHomePage` in `main.dart` to not be a `Scaffold` to avoid nesting scaffolds